flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/utils/testTableSourceSinks.scala

```diff
@@ -110,26 +111,25 @@ object TestTableSourceSinks {
 
   def createCsvTemporarySinkTable(
       tEnv: TableEnvironment,
-      schema: TableSchema,
+      tableSchema: TableSchema,
       tableName: String,
       numFiles: Int = 1): String = {
-    val tempFile = File.createTempFile("csv-test", null)
-    tempFile.deleteOnExit()
-    val path = tempFile.getAbsolutePath
-
-    val sinkOptions = collection.mutable.Map(
-      "connector.type" -> "filesystem",
-      "connector.path" -> path,
-      "format.type" -> "csv",
-      "format.write-mode" -> "OVERWRITE",
-      "format.num-files" -> numFiles.toString
-    )
-    sinkOptions.putAll(new Schema().schema(schema).toProperties)
-
-    val sink = new CsvBatchTableSinkFactory().createStreamTableSink(sinkOptions);
-    tEnv.asInstanceOf[TableEnvironmentInternal].registerTableSinkInternal(tableName, sink)
-
-    path
+    val tempDir = Files.createTempDirectory("csv-test").toFile
+    tempDir.deleteOnExit()
+    val tempDirPath = tempDir.getAbsolutePath
+
+    Files.createTempDirectory("csv-test")
+    val schema = tableSchema.toSchema
+    val tableDescriptor = TableDescriptor
```

Comment:
You have remove the `format.write-mode` option, what is the write mode now, `insert into` or `insert overwrite`?

Reply:
The old CSV sink writes to a specified file, so it needs to distinguish between `append` and `override` write modes. In contrast, the new filesystem connector writes to a directory, so it does not need to be aware of `append` or `override` write modes.